cmd/compute-domain-kubelet-plugin/driver.go

```diff
@@ -65,36 +67,40 @@ func NewDriver(ctx context.Context, config *Config) (*driver, error) {
 	}
 	driver.state = state
 
-	plugin, err := kubeletplugin.Start(
+	helper, err := kubeletplugin.Start(
 		ctx,
-		[]any{driver},
+		driver.plugin,
```

Comment:
I'm confused by this API -- is this really how it works now? If `device.plugin` is not a pointer, what effect does passing it in here have?

Reply:
Oh, in that commit that you reviewed here things built. But crashed at runtime.

`driver.plugin` did satisfy the interface. But it was certainly _not_ the right thing to inject here :)

I changed that in a later commit on this branch.

Will squash commits.